tests/unittests/Repro.cpp

```diff
@@ -768,6 +782,14 @@ int run() {
   }
 
   if (glowDumpTrace) {
+    if (glowEnableDeviceTrace) {
+      // Stop device traces and collect events.
+      if (hostManager->stopDeviceTrace()) {
```

Comment:
So if this return true, it means it failed? It's a bit counterintuitive?

Reply:
@yinghai Since the API returns Error object, true means there is an error. 
Would it make more senses/clearer to change it to something like this? :
if (glowEnableDeviceTrace){
      // Stop device traces and collect events.
      Error err = hostManager->stopDeviceTrace();
      if(err){
        LOG(WARNING) << "Failed to stop device traces.";